src/cargo/ops/vendor.rs

```diff
@@ -578,3 +561,25 @@ fn copy_and_checksum<T: Read>(
             .with_context(|| format!("failed to write to {:?}", dst_path))?;
     }
 }
+
+/// Filters files we want to vendor.
+///
+/// `relative` is a path relative to the package root.
+fn vendor_this(relative: &Path) -> bool {
```

Comment:
Should we filter out `.cargo_vcs_info.json` and explicitly bring it back in?

Also, we're closing #11000 without having a test for that case

Reply:
Got it rearranged.